### ISSUE TITLE
Expected absolute ivy home, obtained None

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/bloop#751]
After upgrading to v1.1.1 I started getting the following errors in the Metals test suite.

```
X tests.SbtSlowSuite.supported-scala 49871ms
  org.eclipse.lsp4j.jsonrpc.ResponseErrorException: ivy.home must be absolute: None
```

**Comment by olafurpg** [comment]
From gitter https://gitter.im/scalacenter/bloop?at=5c094b0b8d4f3a2a7c7df707

> Tim Nieradzik @tindzk 17:15
> We tracked it down: The blp-server script contains -Divy.home=None on macOS.

**Comment by jvican** [comment]
A temporary workaround is to remove the `ivy.home` in the `blp-server`, placed in the bloop cellar installation directory:
  
```diff
#!/usr/bin/env sh

BASE_BIN_DIR=$(dirname "$0")
COURSIER_BIN="$BASE_BIN_DIR/blp-coursier"
- /usr/libexec/java_home -v 1.8 -F -R --exec java      -Divy.home=None -jar
+ /usr/libexec/java_home -v 1.8 -F -R --exec java -jar 
"$COURSIER_BIN" launch ch.epfl.scala:bloop-frontend_2.12:1.1.1      -r bintray:scalameta/maven      -r bintray:scalacenter/releases      -r https://oss.sonatype.org/content/repositories/staging      --main bloop.Server
```

**Comment by jvican** [comment]
Fixed in v1.1.1, please upgrade! you may need to delete the cached install.py manually whenever you get an error upgrading.

**Comment by tindzk** [comment]
Thank you for fixing it!
